spec/peep_spec.rb

```diff
@@ -1,22 +1,34 @@
 require 'peep'
-require 'setup_test_database'
+require 'database_helpers'
 
 describe Peep do
 
+
+
   describe '.all' do
     it 'returns a list of peeps' do
-      setup_test_database
-      add_row_to_test_database
+      peep = Peep.create(message: 'This is a peep!')
+      Peep.create(message: 'this is my first peep!')
 
       peeps = Peep.all
-      expect(peeps).to include('This is a peep!')
+
+      expect(peeps.length).to eq 2
+      expect(peeps.first).to be_a Peep
+      expect(peeps.first.id).to eq peep.id
+      expect(peeps.first.message).to eq 'This is a peep!'
+      expect(peeps.first.posted_on).to eq peep.posted_on
```

Comment:
So you'd know if the first peep returned was the same one you added first because it'll be the only one with the string message 'This is a peep!'. So, I don't know if the id test is useful, it seems like we're testing the same thing twice. It is possible it could be useful though, it just feels superfluous.